xformers/components/swiglu/cuda/gemm_fused_operand_sum.cu

```diff
@@ -25,12 +25,8 @@ void gemm_fused_operand_sum_(
   at::cuda::CUDAGuard device_guard(a.device());
   cudaStream_t stream = at::cuda::getCurrentCUDAStream();
 
-  int64_t M = a.size(0);
-  int64_t N = b.size(0);
-  int64_t K = a.size(1);
-
   // templati-ze the cutlass kernel
-  cutlass::gemm::GemmCoord problem_size(M, N, K);
+  cutlass::gemm::GemmCoord problem_size(a.size(0), b.size(1), a.size(1));
```

Comment:
Wait, this is different from what you had before (`b.size(1)` vs `b.size(0)`). Was this a bug before?

Reply:
Looks like this wasn't used before